internal/aws/k8s/k8sclient/clientset.go

```diff
@@ -459,3 +464,42 @@ func (c *K8sClient) Shutdown() {
 		}
 	}
 }
+
+// InClusterConfig is copy of rest.InClusterConfig.
+// There is known bug in InClusterConfig on Windows when running it as host process container.
+// https://github.com/kubernetes/kubernetes/issues/104562
+// This copy fixes that bug by appending `CONTAINER_SANDBOX_MOUNT_POINT` in k8s token and cert file paths.
+// todo: Remove this workaround func when Windows AMIs has containerd 1.7 which solves upstream bug.
+func (c *K8sClient) inClusterConfig() (*rest.Config, error) {
```

Comment:
change the method name as mentioned in comments to InClusterConfig(

Reply:
I see this in comment "Remove this workaround func when Windows AMIs has containerd 1.7 which solves upstream bug." How do we track this in future and make sure its handled ?

Reply:
After discussion with chad, I have cut ticket to my team's CTI on behalf of CW agent team. I will share link to ticket in thread.